src/kernels/exponentiated.jl

```diff
@@ -1,14 +1,15 @@
 """
-`ExponentiatedKernel([ρ=1])`
+    ExponentiatedKernel()`
```

Comment:
```suggestion
    ExponentiatedKernel()
```
I think, we should also consistently use the constructor in the doc strings (e.g., that's not the case for `CosineKernel`).